hazelcast/src/test/java/com/hazelcast/config/alto/AltoSocketConfigTest.java

```diff
@@ -73,42 +62,29 @@ public void testClientPortsNotEnough() {
         assertThrows(HazelcastException.class, () -> createHazelcastInstance(config));
     }
 
+    @Test
+    public void testCreatingHazelcastInstanceThrows_whenPortRangeMatchesButDecreasing() {
```

Comment:
It should fail at making the configuration. Not when making the Hazelcast instance.